sdk/appconfiguration/azure-appconfiguration-provider/samples/refresh_sample.py

```diff
@@ -0,0 +1,64 @@
+# ------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -------------------------------------------------------------------------
+from azure.appconfiguration.provider import load
+from azure.appconfiguration import (
+    AzureAppConfigurationClient,
+    ConfigurationSetting,
+    SentinelKey,
+)
+import os
+import time
+
+connection_string = os.environ.get("AZURE_APPCONFIG_CONNECTION_STRING")
+print(connection_string)
+
+# Setting up a configuration setting with a known value
+client = AzureAppConfigurationClient.from_connection_string(connection_string)
+
+configuration_setting = ConfigurationSetting(key="message", value="Hello World!")
+
+client.set_configuration_setting(configuration_setting=configuration_setting)
+
+
+def my_callback_on_fail():
```

Comment:
Callback takes an exception parameter:
```suggestion
def my_callback_on_fail(error):
```